core-plugins/src/main/java/co/cask/hydrator/plugin/common/FileSetUtil.java

```diff
@@ -83,18 +86,57 @@ public static void configureParquetFileSet(String configuredSchema, FileSetPrope
+   * ORC input and output format, and also configured for Explore to use Hive. The schema is added
+   * to the file set properties in all the different required ways:
+   * <ul>
+   * <li>As a top-level dataset property;</li>
+   * <li>As the schema for the input and output format;</li>
+   * <li>As the schema to be used by the ORC serde (which is used by Hive).</li>
+   * </ul>
+   * @param configuredSchema the original schema configured for the table
+   * @param properties       a builder for the file set properties
+   */
+  public static void configureORCFileSet(String configuredSchema, FileSetProperties.Builder properties)
+    throws IOException, UnsupportedTypeException {
+    //TODO test if complex cases run with lowercase schema only
+    String lowerCaseSchema = configuredSchema.toLowerCase();
+    String hiveSchema = parseHiveSchema(lowerCaseSchema, configuredSchema);
+    hiveSchema = hiveSchema.substring(1, hiveSchema.length() - 1);
+
+    //Convert to ORCSchema
+    co.cask.cdap.api.data.schema.Schema schemaObj = co.cask.cdap.api.data.schema.Schema.parseJson(configuredSchema);
+    StringBuilder builder = new StringBuilder();
+    HiveSchemaConverter.appendType(builder, schemaObj);
+    String orcSchema = builder.toString();
+
+    properties.setInputFormat(OrcInputFormat.class)
+      .setOutputFormat(OrcOutputFormat.class)
+      .setExploreInputFormat("org.apache.hadoop.hive.ql.io.orc.OrcInputFormat")
+      .setExploreOutputFormat("org.apache.hadoop.hive.ql.io.orc.OrcOutputFormat")
+      .setSerDe("org.apache.hadoop.hive.ql.io.orc.OrcSerde")
+      .setExploreSchema(hiveSchema)
+      .setEnableExploreOnCreate(true)
+      .setInputProperty("orc.mapred.output.schema", orcSchema)
+      .setOutputProperty("orc.mapred.output.schema", orcSchema)
+      .build();
+  }
+
+  /**
    * Configure a file set to use Avro file format with a given schema. The schema is parsed
    * as an Avro schema, validated and converted into a Hive schema. The file set is configured to use
    * Avro key input and output format, and also configured for Explore to use Avro. The schema is added
    * to the file set properties in all the different required ways:
    * <ul>
-   *   <li>As a top-level dataset property;</li>
-   *   <li>As the schema for the input and output format;</li>
-   *   <li>As the schema of the Hive table;</li>
-   *   <li>As the schema to be used by the Avro serde (which is used by Hive).</li>
+   * <li>As a top-level dataset property;</li>
+   * <li>As the schema for the input and output format;</li>
+   * <li>As the schema of the Hive table;</li>
+   * <li>As the schema to be used by the Avro serde (which is used by Hive).</li>
    * </ul>
+   *
    * @param configuredSchema the original schema configured for the table
-   * @param properties a builder for the file set properties
+   * @param properties       a builder for the file set properties
```

Comment:
same here. Please make sure your ide is configured with the same conventions as the rest of the project.